arch/SH/SHDisassembler.c

```diff
@@ -99,7 +99,7 @@ static void set_groups(cs_detail *detail, int n, ...)
 	va_start(g, n);
 	while (n > 0) {
 		sh_insn_group grp;
-		grp = va_arg(g, sh_insn_group);
+		grp = va_arg(g, sh_insn_group); // NOLINT(clang-analyzer-valist.Uninitialized)
```

Comment:
```suggestion
		grp = va_arg(g, sh_insn_group);
```

No need to add an exception here. Otherwise we forget about it. In this specific case we just ignore the red workflow for now. And fix it before the release.